zulipterminal/ui_tools/boxes.py

```diff
@@ -918,3 +931,10 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
             self.panel_view.set_focus("body")
             self.panel_view.keypress(size, 'esc')
         return super().keypress(size, key)
+
+
+# Decoration widget that always sets container widget's focus as false.
+# Doesn't have its own keypress method.
+class DisableFocus(urwid.WidgetDisable):
+    def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
+        return key
```

Comment:
This does seem to make it not editable, but tab on the topic seems to put the focus on it nevertheless?

In a similar way, should we be allowing the stream to be edited?